dubbo-samples-rest/README.md

```diff
@@ -28,12 +28,12 @@ We can use two steps to export Rest Service in Dubbo:
 Step 1: Add dependency to the pom file in our project:
 ```
         <dependency>
-            <groupId>org.apache</groupId>
+            <groupId>com.alibaba</groupId>
             <artifactId>dubbo-rpc-rest</artifactId>
             <version>${dubbo.rpc.version}</version>
         </dependency>
 ```
-
+If you are using is [2.6.1~2.6.5] version,then groupId is com.alibaba ,else groupId is org.apache.dubbo .
```

Comment:
pls make it clear that only if dubbo's version is 2.7.0 or above, its groupId is org.apache.dubbo

Reply:
Your statement is more accurate , only 2.6.1  or above exist dubb-rpc-rest ,other versions are not exist .

Reply:
I have made changes according to your suggestion.